pkg/parse/root.go

```diff
@@ -732,7 +732,7 @@ func prependRootSyncRemediatorStatus(ctx context.Context, c client.Client, syncN
 	var errs []v1beta1.ConfigSyncError
 	for _, conflictErr := range conflictErrs {
 		conflictCSEError := conflictErr.ToCSE()
-		conflictPairCSEError := conflictErr.CurrentManagerError().ToCSE()
+		conflictPairCSEError := conflictErr.ToCSE()
 		errorFound := false
 		for _, e := range rs.Status.Sync.Errors {
 			// Dedup the same remediator conflict error.
```

Comment:
It looks like the `if` statement here should change since the error message is now always the same

Reply:
The format is the same, but the inputs are different. The message includes both the old and new manager, which will swap places depending on which reconciler detected the conflict.

Reply:
If that's the intent then it doesn't look like that's what the code here is doing. Both error messages it's comparing in the OR are the same `conflictErr.ToCSE()`

Reply:
Hmm. You're right. Before this PR, you could inverted the error and simulate an error written by another root reconciler, but now you can't. So this isn't actually de-duping those any more, even if it does now de-dupe the applier and remediator errors.

So I guess we still need to be able to invert the message for deduping.

Reply:
Fixed.